test/extended/builds/docker_pullsecret.go

```diff
@@ -35,29 +35,16 @@ var _ = g.Describe("[builds][pullsecret][Conformance] docker build using a pull
 			err := oc.Run("create").Args("-f", buildFixture).Execute()
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			g.By("starting a test build")
-			out, err := oc.Run("start-build").Args("docker-build").Output()
-			fmt.Fprintf(g.GinkgoWriter, "\nstart-build output:\n%s\n", out)
-			o.Expect(err).NotTo(o.HaveOccurred())
-
-			g.By("expecting the build succeeds")
-			err = exutil.WaitForABuild(oc.REST().Builds(oc.Namespace()), "docker-build-1", exutil.CheckBuildSuccessFn, exutil.CheckBuildFailedFn)
-			if err != nil {
-				exutil.DumpBuildLogs("docker-build", oc)
-			}
+			g.By("starting a build")
+			br, err := exutil.RunBuildConfigAndWait(oc, "docker-build")
```

Comment:
> There is a AssertFailure() as well. 

right, but i can't use that with RunStartBuildConfigAndWait because today that function demands success, it doesn't let me evaluate the outcome, it does it for me.